public/locales/fr/email.json

```diff
@@ -32,8 +32,8 @@
   "email-sent-confirmation": "Un courriel a été envoyé s'il y avait une correspondance dans nos dossiers avec le numéro de dossier (FEDS).",
   "email-confirmation-msg": {
     "request-received": "Votre demande pour obtenir le numéro de dossier a été reçue",
```

Comment:
The french translation should have been updated to match the new English content.